src/Source/FileSource.js

```diff
@@ -136,6 +136,14 @@ class FileSource extends Source {
         return this.url;
     }
 
+    onParsedFile(parsedFile) {
+        this.parsedData = parsedFile;
+        if (parsedFile.extent && parsedFile.extent.crs != 'EPSG:4978') {
```

Comment:
I know it's me that put this filter here, but I don't really remember the case where it is necessary. Do you know if it is still necessary ?